packages/villus/src/utils/common.ts

```diff
@@ -1,4 +1,5 @@
 import { getCurrentInstance, inject, InjectionKey, isReactive, isRef, Ref, toRefs, WatchSource } from 'vue';
+import { activeClient, setActiveClient } from '../client';
```

Comment:
Since I mentioned earlier avoiding exporting `activeClient` we can use `getActiveClient` instead since it would be safer if my suggestions are implemented.

Reply:
Yes, I do think it would be better as you suggested